lib/flame.dart

```diff
@@ -1,40 +1,30 @@
 library flame;
 
-import 'package:audioplayers/audio_cache.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 
 import 'assets/assets_cache.dart';
 import 'assets/images.dart';
-import 'bgm.dart';
-import 'flame_audio.dart';
 import 'util.dart';
 
 /// This class holds static references to some useful objects to use in your game.
 ///
-/// You can access shared instances of [AudioCache], [Images] and [Util].
+/// You can access shared instances of [AssetsCache], [Images] and [Util].
 /// Most games should need only one instance of each, and should use this class to manage that reference.
 class Flame {
   // Flame asset bundle, defaults to root
   static AssetBundle _bundle;
   static AssetBundle get bundle => _bundle == null ? rootBundle : _bundle;
 
-  /// Access a shared instance of the [FlameAudio] class.
-  static FlameAudio audio = FlameAudio();
-
-  /// Access a shared instance of the [Bgm] class.
-  static Bgm _bgm;
-  static Bgm get bgm => _bgm ??= Bgm();
+  /// Access a shard instance of [AssetsCache] class.
```

Comment:
I know this was not written by you, but there is a typo here: `shared`

Reply:
boy scout's rule!